src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -1,6 +1,7 @@
 import asyncio
 import logging
 import pickle
+import traceback
```

Comment:
debug leftover

Reply:
traceback is now used to log more info in case something crashed in the evaluator tasks.